### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by adem4ik**

### CODE REVIEW

File: avidemux_ru.ts [hunk 1]
```diff
@@ -7876,7 +7876,7 @@ Start and/or end time would be out of range</source>
     <message>
         <location line="+8"/>
         <source>libavcodec MPEG-2 configuration</source>
-        <translation>Настройки кодировщика libavcodec MPEG-2</translation>
+        <translation>Настройки libavcodec MPEG-2</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts:7879]: Just curious why you drop the word for "encoder" here, but keep it for libaom. For me, both cases are completely identical ("Settings" "encoder" $nameOfTheLibrary $codecName).

File: avidemux_ru.ts [hunk 2]
```diff
@@ -12062,27 +12062,27 @@ The file must be re-indexed. Proceed?</source>
     <message>
         <location line="+828"/>
         <source>Input: %1x%2, %3fps  |  Decoder: %4  |  Display: %5  |  Zoom: %6%</source>
-        <translation type="unfinished"></translation>
+        <translation>Вход: %1x%2, %3к/с  |  Декодер: %4  |  Изображение: %5  |  Масштаб: %6%</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts]: I apologize in advance for probable changes in the status bar code in near future, which will invalidate the translation. It is fine for now, of course.

File: avidemux_ru.ts [hunk 3]
```diff
@@ -723,7 +723,7 @@ Drop timing informations ?</source>
     <message>
         <location line="-988"/>
         <source>Play [SPACE]</source>
-        <translation>Воспроизвести [SPACE]</translation>
+        <translation>Играть [ПРОБЕЛ]</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts]: Please have a look at https://github.com/mean00/avidemux2_i18n/pull/12#discussion_r665725179

File: avidemux_ru.ts [hunk 4]
```diff
@@ -2775,17 +2775,17 @@ Is it running ?</source>
     <message>
         <location line="+1"/>
         <source>Refresh Rate Cap (ms)</source>
-        <translation>Интервал частоты обновления (мс)</translation>
+        <translation>Предел частоты обновления (мс)</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts]: The previous translation was somewhat closer to the real meaning of the spinbox that lets user set the minimum time in milliseconds between two consecutive redraws. The source string is poorly worded, "rate" implies Hz as unit.

File: avidemux_ru.ts [hunk 5]
```diff
@@ -2881,7 +2881,7 @@ Is it running ?</source>
         <location line="+8"/>
         <location line="+2"/>
         <source>Clear recent items</source>
-        <translation>Очистить список</translation>
+        <translation>Очистить список недавних</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts]: Substantive missing? Sounds very broken for me. I believe the previous translation was better.

File: avidemux_ru.ts [hunk 6]
```diff
@@ -11246,7 +11246,7 @@ Only VP8/VP9/AV1 video and Vorbis/Opus audio supported</source>
         <location line="+48"/>
         <source>Cannot get AAC Extra data
 </source>
-        <translation>Невозможно получить дополнительные данные AAC
+        <translation>Невозможно получить расширенные данные AAC
 </translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts:11250]: I believe that "extradata" is either left as-is or translated as in the previous translation. Searching the internet for the submitted updated translation returns hardly any results (at the moment none at all).

This applies to all instances of translation of "extradata" below.

**Review by eumagga0x2a — COMMENTED** [review]
Thank you, I've added a few remarks.

File: avidemux_ru.ts (hunk 4 above)
Comment (adem4ik) [reply avidemux_ru.ts]: But "интервал частоты" (rate interval) is meaningless tbh. I'll change that string to "период обновления" (refresh interval).

File: avidemux_ru.ts (hunk 1 above)
Comment (adem4ik) [reply avidemux_ru.ts:7879]: I've found that some other similar strings without "encoder" don't have "кодировщик". So I decided to keep the consistency.
изображение
изображение

File: avidemux_ru.ts (hunk 6 above)
Comment (adem4ik) [reply avidemux_ru.ts:11250]: What about "экстраданные"? It is direct translation and there are some relevant results in Google and Yandex.

File: avidemux_ru.ts (hunk 5 above)
Comment (adem4ik) [reply avidemux_ru.ts]: Okey, I'll revert it.

File: avidemux_ru.ts (hunk 1 above)
Comment (eumagga0x2a) [reply avidemux_ru.ts:7879]: I assume that "Encoder" in old source strings was omitted for brevity, but the result sounds horrible for me now. I'm inclined to fix them all at some point.

File: avidemux_ru.ts (hunk 6 above)
Comment (eumagga0x2a) [reply avidemux_ru.ts:11250]: The difficulty to find an appropriate translation demonstrates how lousy the English term for this amalgamation of decoder instructions and display hints is. Not really happy with "экстраданные", but unable to provide a convincing alternative either.

**Review by eumagga0x2a — COMMENTED** [review]
Thank you for the update, I'll merge it.

**Pull request merged**